vignettes/setup-gcp.Rmd

```diff
@@ -72,14 +72,14 @@ Once you have the client Id JSON file, you should be able to use `cr_setup()` or
 
 To enable automatic provision of the service accounts are services you may also need to active the APIs necessary
 
-* IAM API `https://console.developers.google.com/apis/api/iam.googleapis.com/overview`
-* Cloud Resource Manager `https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview`
-* Cloud Run `https://console.developers.google.com/apis/api/run.googleapis.com/overview`
-* Cloud Build `https://console.cloud.google.com/apis/api/cloudbuild.googleapis.com/overview`
```

Comment:
I do this to get the URLs through CRAN checks which sometimes trip over redirects etc. Although may not be necessary if the vignettes are not being built for CRAN